config/mocks/wallet-options-v4.json

```diff
@@ -32,6 +32,7 @@
         }
       },
       "featureFlags": {
+        "applePayPaymentMethod": true,
```

Comment:
don't forget to update this feature flag in the applications projects as well!

Reply:
When you come with the flour, I already have the cake! hahaha

https://github.com/blockchain/applications/pull/11360